src/app/core/config/config-fix.ts

```diff
@@ -714,6 +714,15 @@ export const defaultJsonConfig = {
               "title": "",
               "component": "EducationalMaterial"
             }
+          ],
+          "summary": [
+            {
+              "Total": $localize `:Total item Count:Total`
+            },
+
+            {
+              "Average": $localize `:Average Per Item:Average`
+            }
```

Comment:
It looks to me like the localized values assigned to Total and Average here are not actually used in the html template? Maybe the config format could be even more simple, just a boolean to turn on/off then?
`summary: { total: true, average: false }`

Reply:
I've changed it based on your suggestions.